src/server_requests.js

```diff
@@ -111,6 +111,44 @@ export function fetchDatastackFromFile(payload) {
   );
 }
 
+/**
+ * Get a list of the column names of a vector file.
+ *
+ * @param {string} payload - path to file
+ * @returns {Promise} resolves array
+ */
+export async function getVectorColumnNames(payload) {
+  const result = await (

```

Comment:
I did this originally because the request returns a JSON `{'colnames': [...]}` and I wanted this function to just return the list `[...]`. Otherwise, functions further up would have to know that the property is called `colnames` to access it. But I just learned that JSON can start with brackets so it'll make more sense to use a list rather than an object.